analytics/config.py

```diff
@@ -11,20 +11,31 @@
     with open(CONFIG_FILE) as f:
         config = json.load(f)
 else:
-    print('Config file %s doesn`t exist, using defaults' % CONFIG_FILE)
+    print('Config file %s doesn`t exist, using environment variables / defaults' % CONFIG_FILE)
 
 
-def get_config_field(field: str, default_val = None):
+def get_config_field(field: str, default_val = None, allowed_values = []):
+    value = None
     if field in os.environ:
-        return os.environ[field]
+        value = os.environ[field]
+    elif config_exists and field in config and config[field] != '':
+        value = config[field]
+    elif default_val is not None:
+        value = default_val
 
-    if config_exists and field in config and config[field] != '':
-        return config[field]
+    if len(allowed_values) > 0 and value not in allowed_values:
+        raise Exception('{} value must be one of: {}, got: {}'.format(field, allowed_values, value))
 
-    if default_val is not None:
-        return default_val
+    if value == None:
```

Comment:
https://www.python.org/dev/peps/pep-0008/#programming-recommendations
```suggestion
    if value is None:
```